packages/teleterm/src/ui/Tabs/TabItem.tsx

```diff
@@ -107,16 +112,16 @@ const StyledTabItem = styled.div(({ theme, active, dragging, canDrag }) => {
   return styles;
 });
 
-const StyledTabButton = styled.button`
-  display: flex;
+const Title = styled(Text)`
```

Comment:
What was the reason behind changing this to a div from a button? 🥺

Reply:
This element ins't clickable, so there is no need for it to be a `button`.
We should think how to change the parent `div` (which has handlers) to be an interactive element.